packages/fxa-auth-server/lib/routes/attached-clients.js

```diff
@@ -25,14 +26,20 @@ module.exports = (log, db, devices, clientUtils) => {
         },
         description: '🔒 sessionToken',
         notes: [
-          '<p>🔒 HAWK-authenticated with session token.</p>' +
-            "<p>Returns an array listing all the clients connected to the authenticated user's account, including devices, OAuth clients, and web sessions.</p>" +
-            `<p>This endpoint is primarily designed to power the "devices and apps" view on the user's account settings page. Depending on the type of client, it will have at least one and possibly several of the following properties:</p>` +
-            '<ul><li><code>clientId</code>: The OAuth client_id of the connected application.</li>' +
-            '<li><code>sessionTokenId</code>: The id of the <code>sessionToken</code> held by that client, if any.</li>' +
-            '<li><code>refreshTokenId</code>: The id of the OAuth <code>refreshToken</code> held by that client, if any.</li>' +
-            "<li><code>deviceId</code>: The id of the client's device record, if it has registered one.</li></ul>" +
-            '<p>These identifiers can be passed to <a href="https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#post-accountattached_clientdestroy"><code>/account/attached_client/destroy</code></a> in order to disconnect the client.</p>',
+          dedent`
+            🔒 HAWK-authenticated with session token.
+
+            Returns an array listing all the clients connected to the authenticated user's account, including devices, OAuth clients, and web sessions.
+
+            This endpoint is primarily designed to power the "devices and apps" view on the user's account settings page. Depending on the type of client, it will have at least one and possibly several of the following properties:
+
+            - <code>clientId</code>: The OAuth client_id of the connected application.
```

Comment:
Fun w/ markdown within template strings! I _think_ we can do backslash and backticks to use markdown code blocks instead of using the `<code>` tags.

 ```- \`clientId\`: The OAuth \`client_id\` of the connected aplication.```

Or, at least that worked in my sloppy test case w/ markdown-it and dedent locally:

```js
const dedent = require("dedent");
const markdownIt = require("markdown-it");

const md = markdownIt({
  html: true,
  linkify: true,
}).disable("code");

const res = md.render(dedent`
  it was the best of times.
  - it was the second best of times.
  - second bullet
    - nested bullet?
      - nested nested bullet?

  there were more times
  but were they \`coded\` times?
`);
console.log(res);
```

Reply:
Thanks @pdehaan - I'll make these updates now!